components/collapsible-panel/collapsible-panel.js

```diff
@@ -178,15 +178,24 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 				margin-inline-start: 0.75rem;
 			}
 
-			.d2l-collapsible-panel.focused {
-				outline: var(--d2l-collapsible-panel-focus-outline);
-			}
+			.d2l-collapsible-panel.focused,
 			:host([expanded]) .d2l-collapsible-panel.focused .d2l-collapsible-panel-header {
 				outline: var(--d2l-collapsible-panel-focus-outline);
 			}
+			@supports selector(:has(a, b)) {
+				.d2l-collapsible-panel.focused,
+				:host([expanded]) .d2l-collapsible-panel.focused .d2l-collapsible-panel-header {
+					outline: none;
+				}
+				.d2l-collapsible-panel.focused:has(:focus-visible),
+				:host([expanded]) .d2l-collapsible-panel.focused:has(:focus-visible) .d2l-collapsible-panel-header {
+					outline: var(--d2l-collapsible-panel-focus-outline);
+				}
+			}
 			:host([expanded]) .d2l-collapsible-panel {
 				outline: none;
 			}
```

Comment:
Previously clicking the header to expand/collapse wouldn't apply the outline, whereas now it does. Is that expected?

Reply:
I think we're observing the browser's initial focus-visible state here. The header itself isn't focusable, just the opener, so clicking it doesn't trigger the browser to set it's focus-visible bit. If we click something else to apply focus (like the language select, the fullscreen switch, or the actual button containing the icon) then we see the expected behaviour. I can run this by Jeff.

Reply:
Ok yep. Ran this by Jeff and he's ok with it.

Note: might go without saying, but it the reason it wasn't getting the outline before was because it wasn't being focused at all when the header was clicked, and that's why `[enter]`/`[space]` wasn't working as he wanted.